config/routes.rb

```diff
@@ -390,6 +390,9 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
+      get '/opt_in_ipp' => 'opt_in_ipp#show'
+      put '/opt_in_ipp' => 'opt_in_ipp#update'
```

Comment:
todo: this page should have a better route. Maybe something like `/in_person/getting_started`